sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -85,15 +86,17 @@ public HttpLoggingPolicy(HttpLogDetailLevel detailLevel, boolean prettyPrintJSON
                 logger.info("(empty body)");
                 logger.info("--> END {}", request.httpMethod());
             } else {
-                boolean isHumanReadableContentType = !"application/octet-stream".equalsIgnoreCase(request.headers().value("Content-Type"));
+                boolean isHumanReadableContentType =
+                    !"application/octet-stream".equalsIgnoreCase(request.headers().value("Content-Type"));
                 final long contentLength = getContentLength(request.headers());
 
                 if (contentLength < MAX_BODY_LOG_SIZE && isHumanReadableContentType) {
                     try {
                         Mono<byte[]> collectedBytes = FluxUtil.collectBytesInByteBufferStream(request.body());
                         reqBodyLoggingMono = collectedBytes.flatMap(bytes -> {
                             String bodyString = new String(bytes, StandardCharsets.UTF_8);
-                            bodyString = prettyPrintIfNeeded(logger, request.headers().value("Content-Type"), bodyString);
+                            bodyString = prettyPrintIfNeeded(logger, request.headers().value("Content-Type"),
+                                bodyString);
```

Comment:
This kind of wrapping makes me a little sad. It makes the code harder to read, in my humble opinion, for no real gain. Can we make the rules different: set a higher max length for code (e.g. 160, 180, even 200 chars), and keep the JavaDoc length at 120? Paging @conniey for her thoughts.

Reply:
Yeah, generally I would like to see some of the weirder splits fixed up to be more sane.